src/hotspot/cpu/aarch64/interp_masm_aarch64.cpp

```diff
@@ -393,7 +393,13 @@ void InterpreterMacroAssembler::dispatch_base(TosState state,
                                               bool verifyoop,
                                               bool generate_poll) {
   if (VerifyActivationFrameSize) {
-    Unimplemented();
+    Label L;
+    sub(rscratch2, rfp, esp);
+    unsigned char min_frame_size = (frame::link_offset - frame::interpreter_frame_mirror_offset) * wordSize;
+    cmp(rscratch2, min_frame_size);
```

Comment:
```suggestion
    int min_frame_size = (frame::link_offset - frame::interpreter_frame_mirror_offset) * wordSize;
    subs(rscratch2, min_frame_size);
```
The use of `subs` here is a bit odd, but it's less odd than defining `min_frame_size` as unsigned char.